python/tvm/_ffi/runtime_ctypes.py

```diff
@@ -58,7 +58,8 @@ class DataType(ctypes.Structure):
         0 : 'int',
         1 : 'uint',
         2 : 'float',
-        4 : 'handle'
+        4 : 'handle',
+        65: 'bfloat'
```

Comment:
use a code that is consistent with dlpack

Reply:
@tqchen Hi, I found an inconsistency here: in Python binding, 4 is mapped to 'handle', while in C++ code, 4 is for Nullptr and 3 is for handle. Is this expected or a bug?

Reply:
The definition of DataType code and PackedFunc argument type code starts to diverge after 3

Reply:
May I ask what I am supposed to do here? In the newest commit, I changed `4 : 'handle'` here to 3, and let 4 mapped to bfloat. I also changed `class TypeCode(object):` and relavent C++ class to let `NULL` mapped to 21, instead of 4. I have noticed that in `class TypeCode(object):`, we have to define both `BFLOAT` and `NULL`, so I thought we needed to resolve the conflict as they were all mapped to 4.

Reply:
4 is the type code of bfloat in dlpack